packages/base64/package.json

```diff
@@ -34,6 +34,8 @@
   },
   "dependencies": {},
   "devDependencies": {
+    "@agoric/eslint-config": "^0.3.6",
+    "@agoric/eslint-plugin": "^0.3.2",
```

Comment:
In follow-up, I hope to change eslint-config such that it depends on the corresponding eslint-plugin as a dependency instead of a devDependency to eliminate this redundancy, if it works that way.